IntroducingMethods/src/cst1201/IntroducingMethods.java

```diff
@@ -5,31 +5,47 @@
 /**
  * TODO: Description of your class.
  *
- * @author TODO: Your name here.
+ * @author TODO: razia and jean.
  */
 public class IntroducingMethods {
 
     public static void main(String[] args) {
 
-        // TODO: Fill in the body with your code
+        System.out.print(" Please enter a string: ");
```

Comment:
This should be inside `getInputString()`. See https://github.com/cst1201-d644/Introducing-Methods/blob/solution/IntroducingMethods/src/cst1201/IntroducingMethods.java.